src/calendar.js

```diff
@@ -441,13 +441,13 @@ function calendar (calendarOptions) {
     return api;
   }
 
-  function setValue (value, force = false) {
+  function setValue (value, force) {
     var date = parse(value, o.inputFormat);
     if (date === null) {
       return;
     }
 
-    if (force) {
+    if (force || false) {
```

Comment:
You can just remove `|| false` here and it's equivalent.

Reply:
fixed